runtime/js/90_deno_ns.js

```diff
@@ -31,6 +31,17 @@ import * as kv from "ext:deno_kv/01_db.ts";
 import * as cron from "ext:deno_cron/01_cron.ts";
 import * as webgpuSurface from "ext:deno_webgpu/02_surface.js";
 
+class FsFile extends fs.FsFile {
```

Comment:
I'm assuming the warnings are here because its easier to remove then at once?

Reply:
Other APIs within the codebase call `fs.FsFile` internally/directly. This class here is only for `Deno.FsFile`.